opba-banking-protocol-facade/src/test/java/de/adorsys/opba/protocol/facade/services/ais/ServiceContextProviderTest.java

```diff
@@ -28,7 +27,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ActiveProfiles("test")
-@SpringBootTest(classes = ServiceContextProviderTest.TestConfig.class)
+@SpringBootTest(classes = ApplicationTest.class)
```

Comment:
You need to add test that validates you have really 'encrypted' - unreadable data in database and in returns

Reply:
done